From 4cd42774d0e568808fc7c0dc09a0ecaceabcb748 Mon Sep 17 00:00:00 2001
From: David Cohen <david.a.cohen@intel.com>
Date: Mon, 4 Aug 2014 11:03:49 -0700
Subject: [PATCH 1/2] usb: dwc3: do not shutdown phy during suspend

phy-generic's init/shutdown is not intended to be used as suspend/resume for
TUSB1210. Let's avoid it for now while a proper suspend/resume is implemented.

Signed-off-by: David Cohen <david.a.cohen@intel.com>
Signed-off-by: Eric Ernst <eric.ernst@intel.com>
---
 drivers/usb/dwc3/core.c |    6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/usb/dwc3/core.c b/drivers/usb/dwc3/core.c
index 7767aad380eb..73f3195c1330 100644
--- a/drivers/usb/dwc3/core.c
+++ b/drivers/usb/dwc3/core.c
@@ -765,9 +765,6 @@ static int dwc3_suspend(struct device *dev)
 	dwc->gctl = dwc3_readl(dwc->regs, DWC3_GCTL);
 	spin_unlock_irqrestore(&dwc->lock, flags);
 
-	usb_phy_shutdown(dwc->usb3_phy);
-	usb_phy_shutdown(dwc->usb2_phy);
-
 	return 0;
 }
 
@@ -776,9 +773,6 @@ static int dwc3_resume(struct device *dev)
 	struct dwc3	*dwc = dev_get_drvdata(dev);
 	unsigned long	flags;
 
-	usb_phy_init(dwc->usb3_phy);
-	usb_phy_init(dwc->usb2_phy);
-
 	spin_lock_irqsave(&dwc->lock, flags);
 
 	dwc3_writel(dwc->regs, DWC3_GCTL, dwc->gctl);
-- 
1.7.9.5

